cypress/integration/granules_spec.js

```diff
@@ -376,6 +376,27 @@ describe('Dashboard Granules Page', () => {
       cy.get('.table .tbody .tr').should('have.length', 2);
     });
 
+    it('Should reingest a granule from granule details page.', () => {
+      const granuleId = 'MOD09GQ.A9344328.K9yI3O.006.4625818663028';
+      cy.intercept(
+        { method: 'PUT', url: new RegExp('/granules/.*') },
+        { body: { message: 'ingested' }, statusCode: 200 },
+      );
+      cy.intercept('GET', `/granules/${granuleId}*`).as('getGranule');
+
+      cy.visit(`/granules/granule/${granuleId}`);
+      cy.wait('@getGranule');
+      cy.get('.heading--large').should('have.text', `Granule: ${granuleId}`);
+      cy.contains('button', 'Options').click();
+      cy.get('.dropdown__menu').contains('Reingest').click();
+      cy.get('.modal-body .form__dropdown .dropdown__element input').as('workflow-input');
+      cy.get('@workflow-input').click({ force: true }).type('IngestAndPublish').type('{enter}');
```

Comment:
Is this a good test?  I can type "anything" in the box and then hit confirm/select and get a success.  I'm 99% sure it's just not selecting anything when I do that.

Reply:
I don't know if that's a bug or feature. 🤷

Reply:
Updated tests to verify that a correct value is selected from dropdown.
[d816123](https://github.com/nasa/cumulus-dashboard/pull/986/commits/d816123c40054dc268dce769b0cc0bbffcdbfdf7)